### ISSUE TITLE
Sign in: Remove sign in types from VA.gov

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#98279]
## Description
Based on our audit in #98274, we need to remove mentions of sign in types on all authenticated and unauthenticated pages on VA.gov.

### Authenticate pages
- [ ] TK: Draft recommendations (with support from MZ, as needed)
- [ ] LW: Review recommendations
- [ ] AP: Share recommendations with product teams

### Unauthenticated pages
- [ ] TK: Draft updates (with support from MZ, as needed)
- [ ] LW: Review updates
- [ ] AP: Make updates in Drupal (while sharing screen with TK, as needed)
- [ ] MZ: Copyedit and publish

## Criteria
- [ ] All content recommendations for authenticated pages have been shared with the appropriate product teams
- [ ] All content updates for unauthenticated pages have been published on VA.gov

**Comment by megzehn** [comment]
Just noting here that we're drafting updates and notes on specific pages [in this Sharepoint tracker](https://dvagov.sharepoint.com/:x:/s/SitewideCAIA/EVCd3gQbVGxAnpg4Ng0iDdEBA26ootEzF_zBnRA0kbrLTg?e=ow7Mhm)

**Comment by megzehn** [comment]
Hi @saratorres2! 

These 3 pages are ready for you to copy edit and publish as soon as you have time: 

- Check your remaining Post-9/11 GI Bill benefits
- COVID-19 testing at VA
- Coronavirus FAQs: What Veterans need to know

This is ready for copy edit, but let's hold until next week after we get Danielle's sign off: 

- COVID-19 vaccines at VA

The rest of the [posts in the sign-in tracker](https://dvagov.sharepoint.com/:x:/s/SitewideCAIA/EVCd3gQbVGxAnpg4Ng0iDdEBA26ootEzF_zBnRA0kbrLTg?e=ow7Mhm) are ready for you to copy edit but we should leave in drafts. Danielle will publish on Feb. 1.

@moviedosoco - we'll need to update the Spanish version of these pages as well. I've noted in the [sign-in tracker](https://dvagov.sharepoint.com/:x:/s/SitewideCAIA/EVCd3gQbVGxAnpg4Ng0iDdEBA26ootEzF_zBnRA0kbrLTg?e=ow7Mhm)  which posts have translations. The edits are all just deletions. There's no extra copy. Are you able to make those edits? Let me know if there are any issues.

**Comment by aprocik1** [comment]
@moviedosoco, I don't think so. We're using this single ticket to track all the necessary updates for removing sign in types on various pages. If you'd prefer, or if the updates to Spanish pages require additional steps not captured here, I could open a new ticket. But, since this is simply removing content, I don't think these updates will need to go to State. Correct me if I'm wrong!

**Comment by moviedosoco** [comment]
@aprocik1 Thank you, A! I noticed that the [Signing in to VA.gov](https://www.va.gov/resources/signing-in-to-vagov/) English page has updated content in Drupal draft, specifically in H2 "Issues with signing in", which has 4 more accordions added. 
We are going to need a separate ticket for this translation work.

**Comment by megzehn** [comment]
Hi @moviedosoco, I just added [this ticket (100997)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/98223?issue=department-of-veterans-affairs%7Cva.gov-team%7C100997) for translating the **Signing in to VA.gov** page. (@aprocik1 let me know if I did anything wrong with that ticket!)

**Comment by saratorres2** [comment]
The following pages have been copy edited and published. 

- Check your remaining Post-9/11 GI Bill benefits 
- COVID-19 testing at VA 
- Coronavirus FAQs: What Veterans need to know
- COVID-19 vaccines at VA 
- Preguntas frecuentes sobre el coronavirus: lo que necesitan saber los veteranos

Outstanding queries are noted below: 

- Some of these pages have a a link to public health info. These links are loops that needs to be fixed.
- Some of these pages still have references or links to My HealtheVet pages or site.

The rest of the [posts in the sign-in tracker](https://dvagov.sharepoint.com/:x:/s/SitewideCAIA/EVCd3gQbVGxAnpg4Ng0iDdEBA26ootEzF_zBnRA0kbrLTg?e=ow7Mhm) are ready for you to copy edit but we should leave in drafts. Danielle will publish on Feb. 1.

**Comment by moviedosoco** [comment]
@aprocik1, @RLHecht The **Vacunas contra el COVID-19 en VA** Spanish page has been updated and is saved in draft.

Drupal nodes:
https://prod.cms.va.gov/node/16359/latest
https://preview-prod.vfs.va.gov/preview?nodeId=16359

Thank you!

**Comment by megzehn** [comment]
Noting that the pages in the sign-in tracker labeled as "Blocked" are ready for @DanielleThierryUSDSVA to do a final review and publish on 2/1.

- [How to access My HealtheVet on VA.gov](https://prod.cms.va.gov/node/69150/edit)
- [Can I still use a DS Logon or My HealtheVet account to sign in?](https://prod.cms.va.gov/node/51919/edit)
- [Signing in to VA.gov](https://prod.cms.va.gov/node/8222/edit)
- [What if I can’t sign in to VA.gov because my password doesn’t work?](https://prod.cms.va.gov/node/8214/)
- [How do I sign in to VA.gov?](https://prod.cms.va.gov/node/8208)

**Comment by RLHecht** [comment]
Danielle published these final changes on 3/5. Will validate and close ticket once changes are live.

**Comment by RLHecht** [comment]
We made the changes to these translated pages as well. Going to close this ticket.
